testSetup.js

```diff
@@ -5,3 +5,8 @@ global.window.URL = {
     return 'data://' + arg.name;
   }
 };
+
+global.window.DataTransferItem = function DataTransferItem(arg) {
```

Comment:
It doesn't look like you need to expose this globally. Why not just use this function in test?